plugins/package-managers/node/src/main/kotlin/Npm.kt

```diff
@@ -281,7 +281,9 @@ open class Npm(
         if (hasIncompleteData) {
             runCatching {
                 getRemotePackageDetails(workingDir, "$rawName@$version")
-            }.onSuccess { details ->
+            }.onFailure { e ->
```

Comment:
Maybe extend the commit message a bit by saying: "Prepare for an upcoming change that signals that parsing of `PackageJson` data was unsuccessful."